Source/ThirdParty/libwebrtc/Source/webrtc/sdk/objc/components/video_codec/RTCVideoEncoderH264.mm

```diff
@@ -473,6 +479,9 @@ - (NSInteger)encode:(RTCVideoFrame *)frame
     codecSpecificInfo:(nullable id<RTCCodecSpecificInfo>)codecSpecificInfo
            frameTypes:(NSArray<NSNumber *> *)frameTypes {
   if (!_callback || ![self hasCompressionSession]) {
+    if (_errorCallback) {
+      _errorCallback(-1);
```

Comment:
It would be nice to have a named constant for “-1”

Reply:
Done